src/components/new-request/business-lookup.vue

```diff
@@ -10,12 +10,12 @@
       append-icon="mdi-magnify"
       autocomplete="chrome-off"
       autofocus
-      class="mt-5 mb-n2"
       filled
-      hint="For example: &quot;Joe's Plumbing Inc.&quot;, &quot;BC1234567&quot;, &quot;FM1234567&quot;"
+      hint="Search by name, incorporation or registration number of existing business"
       item-text="name"
       item-value="identifier"
-      label="Business Name or Incorporation/Registration Number"
+      label="Find an existing business"
+      hide-details="auto"
```

Comment:
As per uxpin:

![image](https://github.com/bcgov/namerequest/assets/10002889/838b065f-a4d7-4c1b-a950-30df9d124289)

PS The `hide-details` prop reduces the height if there is no message. I did this elsewhere, too.